python/ray/tests/test_autoscaler.py

```diff
@@ -414,6 +420,15 @@ def testGetOrCreateHeadNode(self):
         runner.assert_has_call("1.2.3.4", "head_setup_cmd")
         runner.assert_has_call("1.2.3.4", "start_ray_head")
         self.assertEqual(self.provider.mock_nodes[0].node_type, None)
+        runner.assert_has_call("1.2.3.4", pattern="docker run")
+        runner.assert_not_has_call(
+            "1.2.3.4", pattern="-v /tmp/ray_tmp_mount/~/ray_bootstrap_config")
```

Comment:
Is there a space missing here (i.e. `-v /tmp/ray_tmp_mount/ ~/ray_bootstrap_config`)

Reply:
Nope, it mounts to `/tmp/ray_tmp_mount/~/ray_bootstrap_config`, no space

Reply:
Ok, sounds good, somehow I'm confused what the `~` is doing here, have never seen it used like this

Reply:
It's because the implementation just appends `/tmp/ray_tmp_mount` to the desired config. I agree it is a bit awkward for `"~"` locations.